src/index.rs

```diff
@@ -641,4 +670,46 @@ mod test {
 			}
 		}
 	}
+
+	#[cfg(feature = "bench")]
+	fn bench_find_entry_internal<
+		F: Fn(&IndexTable, u64, usize, &[u8; CHUNK_LEN]) -> (Entry, usize),
+	>(
+		b: &mut Bencher,
+		f: F,
+	) {
+		let table =
+			IndexTable { id: TableId(18), map: RwLock::new(None), path: Default::default() };
+		let mut chunk = [0u8; 512];
+		let mut keys = [0u64; 64];
+		let mut rng = rand::thread_rng();
```

Comment:
Not too sure if we should bench on random content (could be just random from a fix seed).

Reply:
OK, even though It does not matter too much what the entry bits are, as long as they are different.